amr-wind/utilities/tagging/CurvatureRefinement.H

```diff
@@ -40,7 +40,7 @@ private:
 
     amrex::Vector<amrex::Real> m_curv_value;
 
-    int m_max_lev_field;
+    size_t m_max_lev_field;
```

Comment:
I think this should be an int and      
`   size_t fcount = std::min(curv_value.size(), m_curv_value.size());
` 
in CurvatureRefinement.cpp should be fixed instead

Reply:
Is changing `fcount` to `int` what you meant?

Reply:
yep, the size of this array will be small so int should be sufficient
`int fcount = static_cast<int> std::min()`